### PR TITLE
tablecodec: correct comments mistake

### PR DESCRIPTION
**Pull request by yuanwhy**

### CODE REVIEW

File: tablecodec/bench_test.go [hunk 1]
```diff
@@ -30,7 +30,7 @@ func BenchmarkEncodeEndKey(b *testing.B) {
 	}
 }
 
-// PrefixNext() is slow than using EncodeRowKeyWithHandle.
+// BenchmarkEncodeRowKeyWithPrefixNex PrefixNext() is slow than using EncodeRowKeyWithHandle.
```
Comment (zhexuany) [inline tablecodec/bench_test.go]: Please add description of BenchmarkEncodeRowKeyWithPrefixNex.

File: tablecodec/tablecodec_test.go [hunk 2]
```diff
@@ -33,7 +33,7 @@ var _ = Suite(&testTableCodecSuite{})
 
 type testTableCodecSuite struct{}
 
-// TODO: add more tests.
+// TestTableCodec TODO: add more tests.
```
Comment (zhexuany) [inline tablecodec/tablecodec_test.go]: Please add description for TestTableCodec

**Comment by shenli** [comment]
LGTM @yuanwhy Thanks!
@zhexuany @coocood PTAL

**Comment by coocood** [comment]
LGTM

**Pull request merged**